src/WebUI/Components/DeploymentsComponent.tsx

```diff
@@ -44,7 +44,7 @@ export class DeploymentsComponent extends BaseComponent<IDeploymentsComponentPro
 
     private _getDeploymentListView() {
         let renderList:JSX.Element[] =[];
-        DeploymentsComponent._generateDeploymentReplicaSetMap(this.props.deploymentList, this.props.replicaSetList).forEach((entry, index) => {
+        DeploymentsComponent._generateDeploymentReplicaSetMap(this.props.deploymentList, this.props.replicaSetList, this.props.nameFilterKey).forEach((entry, index) => {

```

Comment:
let the filter be part of the deploymentcomponentprop, rather a common property, as in future this may change